zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterResultMessage.java

```diff
@@ -38,6 +50,14 @@ public String getData() {
     return data;
   }
 
+  public List<ColumnDef.TYPE> getListOfColumnTypes() {
+    return msgColumnTypes.getListOfColumnTypes();
+  }
+
+  public MessageColumnTypes getMessageColumnTypes() {
```

Comment:
This looks used only one place inside `RemoteInterpreterServer`. Can we keep only `List<ColumnDef.TYPE>` as a member variable and make `MessageColumnTypes` in RemoteInterpreterServer` instead?

Reply:
I think it's better to keep `MessageColumnTypes` class for better flexibility (e.g. custom `toString` method).

Reply:
Ah... What I meant is not to remove `MessageColumnTypes`. You don’t have to store it in this class by making new class. How about storing List... and making `MessageColumnTypes` if it’s needed.

Reply:
Oh, ok, thx. Done